test/keeper.ts

```diff
@@ -0,0 +1,33 @@
+export interface TestKeys {
+    apiKey: string;
+    testAuthor: string;
+    testAuthorId?: string;
+    testChannel: string;
+    testConversationId?: string;
+    testCommentId?: string;
+    testInbox: string;
+    testInboxId?: string;
+    testMessageId?: string;
+    testMessageConvRef?: string;
+    testMessageResponse?: string;
+    testMessageSubject?: string;
+}
+
+export class VaultKeeper {
+    private testKeys: TestKeys;
+
+    constructor(varName: string) {
+        // Load the test keys
+		this.testKeys = JSON.parse(varName);
```

Comment:
This will fail with a generic "JSON.parse requires at least one parameter" (on Safari at least) -- you might want to fail earlier (maybe even in `gulpfile.js`?) with a nice error message if `FRONT_TEST_KEYS` isn't set.